packages/shared/src/server/services/SlackService.ts

```diff
@@ -88,9 +88,12 @@ export function parseSlackInstallationMetadata(
  */
 export class SlackService {
   private static instance: SlackService | null = null;
-  private installer: InstallProvider;
+  private readonly installer: InstallProvider;
+  // Total number of records to fetch from Slack
+  private readonly fetchLimit: number;
```

Comment:
Why do you reassign this to a private variable instead of directly consuming the env variable? To me this feels very redundant.